redash/query_runner/databricks.py

```diff
@@ -35,6 +36,53 @@ def _build_odbc_connection_string(**kwargs):
     return ";".join([f"{k}={v}" for k, v in kwargs.items()])
 
 
+def split_sql_statements(query):
+    def strip_trailing_comments(stmt):
+        idx = len(stmt.tokens) - 1
+        while idx >= 0:
+            tok = stmt.tokens[idx]
+            if tok.is_whitespace or sqlparse.utils.imt(tok, i=sqlparse.sql.Comment, t=sqlparse.tokens.Comment):
```

Comment:
will several comment be handled in this case? Like "-- comment1 -- comment2"

Reply:
It may happen only if each comment is on the new line - when they're on the same line it's the single comment with `--` inside of it. The function will handle it (that's why there is a `while` loop 😉 ), I'll add a tests case. Thank you!